packages/d/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
```

Comment:
This should probably be `import * as React from 'react'` 👇 

https://github.com/facebook/react/pull/18102